modules/jabba/1.0/src/R/make_pon.R

```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/env Rscript
+
+script_path <- normalizePath(dirname(unlist(strsplit(commandArgs()[grepl('--file', commandArgs())], split = '='))[2]))
+
+args <- commandArgs(trailingOnly = TRUE)
+
+threads  <- args[1]
+nrml.dir <- args[2]
+tbl.out  <- args[3]
+pon.path <- args[4]
+par.path <- args[5]
+build    <- args[6]
+choose.samples <- args[7]
+
+library(data.table)
```

Comment:
in this and another scripts, I might have missed this, but seems like data.table, dryclean, and parallel are not included in the yaml file. Or they are installed elsewhere (not through environment)?